pkg/aggregator/sender.go

```diff
@@ -129,6 +129,13 @@ func (s *checkSender) SetCheckCustomTags(tags []string) {
 	s.checkTags = tags
 }
 
+// GetCheckCustomTags gets current checkTags of the sender.
+func (s *checkSender) GetCheckCustomTags() []string {
```

Comment:
you should make the function name clear that you are doing a copy of the `sender.checkTags`

```suggestion
func (s *checkSender) GetCopyCheckTags() []string {
```